src/CarbonAware.Aggregators/test/CarbonAwareParametersTests.cs

```diff
@@ -0,0 +1,227 @@
+using CarbonAware.Aggregators.CarbonAware;
```

Comment:
There are a couple of warnings to be addressed

Reply:
They are fake warnings (we do the null check two lines up) but added the `!` just for github cleaness